ecosystem/indexer-grpc/indexer-grpc-table-info/src/table_info_service.rs

```diff
@@ -62,11 +67,28 @@ impl TableInfoService {
                 .await;
             let max_version = self.get_max_batch_version(results).unwrap_or_default();
             let versions_processed = max_version - self.current_version + 1;
+            let context = self.context.clone();
+            let backup_restore_operator = self.backup_restore_operator.clone();
+            let start_version = self.current_version;
+            let indexer_async_v2 = self.indexer_async_v2.clone();
+
+            // Try uploading the rocksdb snapshot by taking a full db checkpoint and save it to gcs if found new epoch
+            // running backup logic in a separate thread to not let it block the main thread to parse table info, since
+            // gcs operation could be slow
+            tokio::spawn(async move {
+                Self::try_backup_db_snapshot(
+                    context.clone(),
+                    max_version,
+                    indexer_async_v2.clone(),
+                    backup_restore_operator.clone(),
+                )
+                .await;
+            });
```

Comment:
is it the same with 
```
            tokio::spawn(
                Self::try_backup_db_snapshot(
                    ...
                )
            });
```
?

Reply:
spawned task is not guaranteed to start immediately, I fear `set_metadata_epoch()` not being called right away will cause duplicated backup tasks.